src/components/atoms/FileInput/FileInput.tsx

```diff
@@ -222,18 +223,19 @@ export default function FileInput<TFile extends UploadFile = UploadFile>({
                 item.data.length > 0 &&
                 item.data.map((dataItem, dataIndex) =>
                   dataItem.type === "password" ? (
-                    <TextInput
-                      onChange={(event) =>
-                        void handleDataChange(event, item, dataIndex)
-                      }
-                      defaultValue={dataItem.value}
-                      innerClassNames={{ input: styles.listItemDataInput }}
-                      key={index}
-                      label={dataItem.label}
-                      placeholder={dataItem.placeholder}
-                      size="small"
-                      type={dataItem.type || "text"}
-                    />
+                    <>
+                      <div>{dataItem.label}</div>
```

Comment:
why are we not passing the label through `label` prop?

Reply:
when we pass prop as label. the whole label gets black background color which is not looking nice. that's qhy i removed label from there and add separate label.